testcases/kernel/syscalls/adjtimex/adjtimex02.c

```diff
@@ -10,115 +10,188 @@
 #include <unistd.h>
 #include <pwd.h>
 #include "tst_test.h"
+#include "lapi/syscalls.h"
 
 #define SET_MODE ( ADJ_OFFSET | ADJ_FREQUENCY | ADJ_MAXERROR | ADJ_ESTERROR | \
-	ADJ_STATUS | ADJ_TIMECONST | ADJ_TICK )
+				ADJ_STATUS | ADJ_TIMECONST | ADJ_TICK )
```

Comment:
This is a useless whitespace change, as a matter of fact the patch is polluted with changes like these and in many cases newly added code has completely wrong indentation. You can use checkpatch.pl shipped with Linux kernel source to check your code for such mistakes.

Reply:
Verified with checkpatch.pl